### PR TITLE
USNP 33 fetching musician (user) data from server

### PR DESCRIPTION
**Pull request by RubySpeeders**

### CODE REVIEW

File: client/src/pages/selectPlayers.tsx [hunk 1]
```diff
@@ -46,7 +38,25 @@ const useStyles = makeStyles(() => ({
...
+    selected: boolean;
+  }
+  interface Branch {
+    branch_id: number;
+    nickname_id: number;
+    branch_name: string;
+  }
+  interface Band {
+    band_id: number;
+    band_name: string;
```
Comment (charlottekies) [inline client/src/pages/selectPlayers.tsx:58]: I would add a branch : Branch property here

File: client/yarn.lock [hunk 2]
```diff
@@ -647,6 +647,11 @@ ast-types-flow@^0.0.7:
   resolved "https://registry.npmjs.org/ast-types-flow/-/ast-types-flow-0.0.7.tgz"
```
Comment (RubySpeeders) [inline client/yarn.lock:647]: I think I put the yarn.lock in the .gitignore in another PR

File: client/src/pages/selectPlayers.tsx [hunk 3]
```diff
@@ -46,7 +38,26 @@ const useStyles = makeStyles(() => ({
   },
 }));
 
-export default function ChoosePlayers() {
+export default function SelectPlayers() {
```
Comment (RubySpeeders) [inline client/src/pages/selectPlayers.tsx:41]: Need help with naming of this page... thoughts?

File: client/src/pages/selectMusicians.tsx [hunk 4]
```diff
@@ -46,7 +38,26 @@ const useStyles = makeStyles(() => ({
   },
 }));
 
-export default function ChoosePlayers() {
+export default function SelectMusicians() {
```
Comment (RubySpeeders) [inline client/src/pages/selectMusicians.tsx:41]: Naming of this page is vexing... thoughts? SelectMusiciansAndInstruments seems too long.

File: client/src/pages/selectPlayers.tsx (hunk 3 above)
Comment (cheesehero112) [reply client/src/pages/selectPlayers.tsx]: maybe something like Dashboard or Main page?

**Comment by cheesehero112** [comment]
@RubySpeeders I run the branch and below screenshot is what I see - which I believe means fetching the musicians are working? I do see the warning about LCP - did you want to look into that? 

Also, do you want to merge the current main branch before PR-ing so that the instrument pictures also show up, or is that something you plan on fetching from the server also?

**Review by cheesehero112 — APPROVED** [review]
Looks great! See the comment/ question above.

**Comment by charlottekies** [comment]
> Also, do you want to merge the current main branch before PR-ing so that the instrument pictures also show up, or is that something you plan on fetching from the server also? 


@cheesehero112 Hi! I have links for instruments too, but I haven't added instruments to the database or an endpoint to use to retrieve them. I am going out of town this week but I'll try to get a task done or two, but for now I'd say continue to retrieve the instruments as usual, and when I get back I'll serve them to you from the backend! 

I found the cause of the crashing I think. More on that later, I'll try to monitor as I'm out.

File: client/src/pages/selectPlayers.tsx (hunk 3 above)
Comment (RubySpeeders) [reply client/src/pages/selectPlayers.tsx]: I went with "Selection" for now; but we can definitely always change it. I am wondering if we could use "Dashboard" for that initial page where users can choose which branch.

**Comment by charlottekies** [comment]
@RubySpeeders @cheesehero112 Chad says no, always different instruments

**Comment by RubySpeeders** [comment]
phew! that's a relief!

**Review by cheesehero112 — APPROVED** [review]
Looks good!

**Pull request merged**